network/p2p/gossip/gossip_test.go

```diff
@@ -611,3 +627,58 @@ type testValidatorSet struct {
 func (t testValidatorSet) Has(_ context.Context, nodeID ids.NodeID) bool {
 	return t.validators.Contains(nodeID)
 }
+
+func TestComputeBloomFilterHitPercentage(t *testing.T) {
```

Comment:
I feel like non-exported code is an implementation detail of the package api so where reasonable I try to test non-exported code through the public api. Since we're already able to test this through AppRequest in the above test, should we just remove this test?

Reply:
Sure will remove it.